app/Http/Controllers/TrainingDash.php

```diff
@@ -734,6 +735,67 @@ public function generateGraphs(Request $request) {
         return $response;
     }
 
+    public function newTrainerFeedback() {
+        $users = User::with('roles')->where('status', '1')->get();
+        $ins = $users->filter(function ($user) {
+            return $user->hasRole('ins');
+        });
+
+        $mtr = $users->filter(function ($user) {
+            return $user->hasRole('mtr');
+        });
+        $feedbackOptions = $ins->merge($mtr);
```

Comment:
3rd time this exists, so let's definitely do this elsewhere and reuse it

Reply:
Done